sormas-backend/src/main/java/de/symeda/sormas/backend/travelentry/travelentryimport/TravelEntryImportFacadeEjb.java

```diff
@@ -0,0 +1,411 @@
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import de.symeda.sormas.api.Language;
+import de.symeda.sormas.api.i18n.Captions;
+import de.symeda.sormas.api.i18n.I18nProperties;
+import de.symeda.sormas.api.i18n.Validations;
+import de.symeda.sormas.api.importexport.ImportLineResultDto;
+import de.symeda.sormas.api.importexport.InvalidColumnException;
+import de.symeda.sormas.api.infrastructure.InfrastructureHelper;
+import de.symeda.sormas.api.infrastructure.PointOfEntryDto;
+import de.symeda.sormas.api.infrastructure.PointOfEntryReferenceDto;
+import de.symeda.sormas.api.person.PersonDto;
+import de.symeda.sormas.api.person.Sex;
+import de.symeda.sormas.api.region.CommunityReferenceDto;
+import de.symeda.sormas.api.region.DistrictReferenceDto;
+import de.symeda.sormas.api.travelentry.DeaContentEntry;
+import de.symeda.sormas.api.travelentry.TravelEntryDto;
+import de.symeda.sormas.api.travelentry.travelentryimport.TravelEntryImportEntities;
+import de.symeda.sormas.api.travelentry.travelentryimport.TravelEntryImportFacade;
+import de.symeda.sormas.api.utils.DateHelper;
+import de.symeda.sormas.api.utils.ValidationRuntimeException;
+import de.symeda.sormas.backend.common.EnumService;
+import de.symeda.sormas.backend.importexport.ImportCellData;
+import de.symeda.sormas.backend.importexport.ImportErrorException;
+import de.symeda.sormas.backend.importexport.ImportFacadeEjb.ImportFacadeEjbLocal;
+import de.symeda.sormas.backend.importexport.ImportHelper;
+import de.symeda.sormas.backend.infrastructure.PointOfEntryFacadeEjb.PointOfEntryFacadeEjbLocal;
+import de.symeda.sormas.backend.person.PersonFacadeEjb.PersonFacadeEjbLocal;
+import de.symeda.sormas.backend.region.CommunityFacadeEjb.CommunityFacadeEjbLocal;
+import de.symeda.sormas.backend.region.DistrictFacadeEjb.DistrictFacadeEjbLocal;
+import de.symeda.sormas.backend.travelentry.TravelEntryFacadeEjb.TravelEntryFacadeEjbLocal;
+import de.symeda.sormas.backend.user.UserService;
+
+@Stateless(name = "TravelEntryImportFacade")
+public class TravelEntryImportFacadeEjb implements TravelEntryImportFacade {
+
+	private final Logger LOGGER = LoggerFactory.getLogger(TravelEntryImportFacadeEjb.class);
+
+	protected static final String ERROR_COLUMN_NAME = I18nProperties.getCaption(Captions.importErrorDescription);
+
+	private static final String PHONE_PRIVATE = "phonePrivate";
+	private static final String PHONE_ADDITIONAL = "phoneAdditional";
+	private static final String EMAIL = "email";
+
+	@EJB
+	private TravelEntryFacadeEjbLocal travelEntryFacade;
+	@EJB
+	private UserService userService;
+	@EJB
+	private ImportFacadeEjbLocal importFacade;
+	@EJB
+	private DistrictFacadeEjbLocal districtFacade;
+	@EJB
+	private CommunityFacadeEjbLocal communityFacade;
+	@EJB
+	private PointOfEntryFacadeEjbLocal pointOfEntryFacade;
+	@EJB
+	private PersonFacadeEjbLocal personFacade;
+
+	@Override
+	public ImportLineResultDto<TravelEntryImportEntities> importData(
```

Comment:
Unit test this...

Reply:
To be perfectly honest, I don't think it's worth writing a unit test at this point because the import logic is more or less a throwaway logic anyway and needs to be rewritten in the future...